src/shared/cloudformation/templateRegistry.ts

```diff
@@ -0,0 +1,106 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import { getLogger } from '../logger/logger'
+import { CloudFormation } from './cloudformation'
+
+export interface TemplateData {
+    templatePath: string
+    templateData: CloudFormation.Template
+}
```

Comment:
Looking for feedback: I like this structure, but I ***hate*** the names (`TemplateData.templateData` = ugh). Got any suggestions?

Reply:
How about `path` and `template`  ?

Reply:
https://i.imgur.com/f5HGxTA.gif